frontend/src/index.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import Menu from './Menu';
```

Comment:
Why did you remove the Menu?

Reply:
because I replace it by App.js (there is the page divided to two parts (header, main) and used little bit different logic for navbar)

Reply:
but I cannot see the Menu from the api call anymore.

Reply:
Just to clarify. We had 3 api calls from the react frontend to our backend (flask or fastAPI). One was the menu, another the events and the other was the about.

Right now I stopped seeing those 3 calls in the code and in the frontend. Can you make sure that we continue having the same 3 calls and we show their results as it was before? Thanks :)

Reply:
Basically when we add functionality we should not remove unrelated functionality. The about call is returning some description and scraping some info from the official pyladies website. We should keep it that way.

The menu might not be so important but again we should not remove functionality in one PR that is not aimed at doing that :) does that make sense?

Reply:
I have added back the events and scraping in react_hooks branch. Please check it.

Reply:
Now I understand why the about, events pages call to backend.  My branch didn't contain this files. Therefore I was confused.

Reply:
I see, you should always make a branch from the latest master :) we can review those things in the next meetup, I think it is useful for everyone :)

Reply:
> I see, you should always make a branch from the latest master :) we can review those things in the next meetup, I think it is useful for everyone :)

definitely but I thought that git branch <name> does it.

Reply:
![git_branch](https://user-images.githubusercontent.com/7299647/97930467-b3ca0700-1d6b-11eb-9b46-e01867eebe43.jpg)

Reply:
I have no idea how you made your branch but if you say some files were missing then you probably did not have the latest master. Even if you created your branch before new commits went into master it is your responsibility to make sure your branch has every commit that is in master :)